concrete-core/src/backends/core/private/crypto/bootstrap/fourier_seeded/mod.rs

```diff
@@ -0,0 +1,302 @@
+use std::fmt::Debug;
+
+use concrete_commons::parameters::{
+    DecompositionBaseLog, DecompositionLevelCount, GlweSize, LweDimension, PolynomialSize,
+};
+use concrete_fftw::array::AlignedVec;
+use serde::{Deserialize, Serialize};
+
+use crate::backends::core::private::{
+    crypto::ggsw::FourierGgswSeededCiphertext,
+    math::{
+        fft::{Complex64, FourierPolynomial},
+        random::RandomGenerator,
+        tensor::{
+            ck_dim_div, AsMutSlice, AsMutTensor, AsRefSlice, AsRefTensor, IntoTensor, Tensor,
+        },
+        torus::UnsignedTorus,
+    },
+};
+
+use super::{FourierBootstrapKey, FourierBuffers, StandardSeededBootstrapKey};
+
+#[cfg(test)]
+mod tests;
+
+/// A bootstrapping key in the fourier domain.
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
+pub struct FourierSeededBootstrapKey<Cont, Scalar>
+where
+    Scalar: UnsignedTorus,
+{
+    // The tensor containing the actual data of the secret key.
+    tensor: Tensor<Cont>,
+    // The size of the polynomials
+    poly_size: PolynomialSize,
+    // The size of the GLWE
+    glwe_size: GlweSize,
+    // The decomposition parameters
+    decomp_level: DecompositionLevelCount,
+    decomp_base_log: DecompositionBaseLog,
+    _scalar: std::marker::PhantomData<Scalar>,
+    seed: u128,
```

Comment:
I ask it here but it counts for every place where a seed is used: should we introduce a `Seed` type? 
(Same goes for shift when it is used, currently it's just a usize.)